README.md

```diff
@@ -10,6 +10,21 @@ The snap can be installed directly from the Stap Store.  Follow the link below f
 
 [![Get it from the Snap Store](https://snapcraft.io/static/images/badges/en/snap-store-black.svg)](https://snapcraft.io/charmed-valkey)
 
+```bash
+sudo snap install charmed-valkey --edge
+```
+
+## Interaction with the snap
+By default, the snap will install and run the valkey-server. You can connect to the server via cli:
```

Comment:
I believe the snap doesn't start the service automatically, until we run the `start` command, right? if yes - we ma need to slightly rephrase this

Reply:
The service is started right after installation:

```
$ sudo snap install charmed-valkey --edge
charmed-valkey (edge) 7.2.5 from Canonical Data Platform (dataplatformbot) installed

$ snap services | grep valkey
charmed-valkey.server              enabled   active    -

$ systemctl status snap.charmed-valkey.server
● snap.charmed-valkey.server.service - Service for snap application charmed-valkey.server
     Loaded: loaded (/etc/systemd/system/snap.charmed-valkey.server.service; enabled; preset: enabled)
     Active: active (running) since Mon 2024-06-17 11:52:46 UTC; 38s ago
[...]

Jun 17 11:52:46 ip-172-31-25-227 systemd[1]: Started snap.charmed-valkey.server.service - Service for snap application charmed-valkey.server.
[...]
Jun 17 11:52:47 ip-172-31-25-227 charmed-valkey.server[5864]: 5864:M 17 Jun 2024 11:52:47.094 * Running mode=standalone, port=6379.
Jun 17 11:52:47 ip-172-31-25-227 charmed-valkey.server[5864]: 5864:M 17 Jun 2024 11:52:47.095 * Server initialized
Jun 17 11:52:47 ip-172-31-25-227 charmed-valkey.server[5864]: 5864:M 17 Jun 2024 11:52:47.100 * Ready to accept connections tcp
```
So I'd leave if that way if you agree.

Reply:
Interesting, but is this intended? We have a [`server` application](https://github.com/canonical/charmed-valkey-snap/blob/adjust-snap-test/snap/snapcraft.yaml#L14-L15) which should be the only way to start the service

Reply:
After discussing - Let's keep it as is until we find a need for delaying the start of the service

Reply:
Yes, both redis and valkey servers can be started right after installation (via the server application, of course).